packages/@react-spectrum/autocomplete/src/SearchAutocomplete.tsx

```diff
@@ -161,7 +164,7 @@ function _SearchAutocompleteBase<T extends object>(props: SpectrumSearchAutocomp
         UNSAFE_className={classNames(styles, 'spectrum-InputGroup-popover', {'spectrum-InputGroup-popover--quiet': isQuiet})}
         ref={popoverRef}
         triggerRef={inputRef}
-        placement={`${direction} end`}
+        placement={`${direction} ${align}`}
```

Comment:
we've changed the default placement from end to start
I think we should set the default to end so we don't move people's alignment on accident
same goes for the rest of them as well

Reply:
You're right. That makes more sense! By the same token though, since default `align` was originally  `start` for `Picker`, I'll only make changes to `SearchAutocomplete` and `ComboBox` (and not to `Picker`).